packages/create-pantheon-decoupled-kit/src/index.ts

```diff
@@ -85,11 +85,34 @@ export const main = async (
 	// If no generators are found in positional params
 	// ask which generators should be run
 	if (!foundGenerators || !foundGenerators.length) {
-		const whichGenerators: QuestionCollection<{ generators: string[] }> = {
+		let generatorsOfCmsType: string[] = [];
+		const cmsType = isString(args.cmsType)
+			? args.cmsType.toLocaleLowerCase()
```

Comment:
```suggestion
			? args.cmsType.toLowerCase()
```
Shouldn't need the Locale distinction here